Gemfile

```diff
@@ -47,6 +46,7 @@ gem 'sentry-sidekiq', '~> 5.18'
 gem 'shakapacker', '6.6.0'
 gem 'sidekiq', '~> 7.2'
 gem 'sidekiq-cron', '~> 1.12'
+gem 'sprockets-rails', '~> 3.5'
```

Comment:
sass-rails must have had this as a dependency and we still need it. Now that active admin and by extension sass is gone, we'll be able to modernise our assets stack and upgrade to propshaft soon ™️